base/stl_helpers.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "std/algorithm.hpp"
+#include "std/auto_ptr.hpp"
```

Comment:
> Could you please point where allocator is used in our project?

I cannot get rid of template template parameters in methods:
```
template<template<class, class> class Cont> void TestSortUnique();
template<template<class, class> class Cont> void TestEqualsBy();
```
Please see my explanation above.